jira/validators/fixversion.groovy

```diff
@@ -16,11 +16,11 @@ if (issue.getFixVersions().isEmpty()) {
 }
 
 // Check that all AffectedVersions are Fixed
-def versions = issue.getAffectedVersions().minus(issue.getFixVersions())
-if (versions.isEmpty()) {
-    log.info("All AffectedVersions are fixed (true)")
+//def versions = issue.getAffectedVersions().minus(issue.getFixVersions())
+//if (versions.isEmpty()) {
+//    log.info("All AffectedVersions are fixed (true)")
     return true
```

Comment:
This line should also be commented out.

Reply:
New suggestion:
```
// Check that all AffectedVersions are Fixed
if (issue.getAffectedVersions().size() <= issue.getFixVersions().size()) {
    log.info("All AffectedVersions are fixed (true)")
    return true
}
```

This will at least make sure that for each affected version we have a fixed version.